code/datums/emotes.dm

```diff
@@ -96,9 +96,10 @@
 	else if(tmp_sound && should_play_sound(user, intentional))
 		if(TIMER_COOLDOWN_CHECK(user, COOLDOWN_MOB_AUDIO))
 			return
+	if(!(HAS_TRAIT(user, TRAIT_EMOTE_CD_EXEMPT)))
 		TIMER_COOLDOWN_START(user, type, audio_cooldown)
 		TIMER_COOLDOWN_START(user, COOLDOWN_MOB_AUDIO, 20 SECONDS)
-		playsound(user, tmp_sound, volume, vary)
+	playsound(user, tmp_sound, volume, vary)
```

Comment:
github sux so i can't do a suggestion here so i'll just paste how it should be done (whitespace included)

```diff
-	if(!(HAS_TRAIT(user, TRAIT_EMOTE_CD_EXEMPT)))
-		TIMER_COOLDOWN_START(user, type, audio_cooldown)
-		TIMER_COOLDOWN_START(user, COOLDOWN_MOB_AUDIO, 20 SECONDS)
-	playsound(user, tmp_sound, volume, vary)
+		if(!HAS_TRAIT(user, TRAIT_EMOTE_CD_EXEMPT))
+			TIMER_COOLDOWN_START(user, type, audio_cooldown)
+			TIMER_COOLDOWN_START(user, COOLDOWN_MOB_AUDIO, 20 SECONDS)
+		playsound(user, tmp_sound, volume, vary)
```